Makefile

```diff
@@ -28,6 +28,9 @@ install-travis:
 	python3.6 -m pip install flake8-mutable flake8-docstrings flake8-builtins flake8-mypy bandit bandit-high-entropy-string
 uninstall:
 	python3.6 -m pip uninstall -y iocage
+	@if [ -f /usr/local/etc/rc.d/ioc ]; then \
+		rm /usr/local/etc/rc.d/ioc; \
+	fi
```

Comment:
how do we ask pip to uninstall our deps?

Reply:
We get rid of all dependencies. 😏Done.

That's not (immediately) possible, so that we will have to figure out if and how other projects do it.